config/locales/models/report.en.yml

```diff
@@ -42,7 +42,7 @@ en:
       budgets: Budgets
       current: Current
       historic: Historic
```

Comment:
Do we need this translation now?

Reply:
I've done a few searches and it looks like it's not used anymore, although of course it's sometimes hard to detect if a string isn't being invoked via interpolation.

I've also renamed some backend code, for consistency and to avoid building up debt with one hand as we remove it with other, if you wouldn't mind having another look to make sure it looks ok?